traveler-swift-core/TravelerKit/Extensions/DateFormatter+PassengerKit.swift

```diff
@@ -46,6 +46,13 @@ extension DateFormatter {
         formatter.dateFormat = "yyyy-MM-dd"
         return formatter
     }()
+
+    public static func dateFormatter(with timeZone: TimeZone) -> DateFormatter {
```

Comment:
This method currently says nothing about the format. Maybe change the caption a bit?